packages/compass-e2e-tests/helpers/commands/close-privacy-settings-modal.js

```diff
@@ -2,23 +2,16 @@ const Selectors = require('../selectors');
 
 module.exports = function (app) {
   return async function closePrivacySettingsModal() {
-    const exists = await app.client.waitForElement(
-      Selectors.PrivacySettingsModal,
-      {
-        mustExist: false,
-        visibleError: 'Expected privacy settings modal to be visible',
-      }
-    );
+    const { client } = app;
 
-    if (!exists) {
+    if (
+      !(await client.existsEventually(Selectors.PrivacySettingsModal, 5000))
```

Comment:
If it exists (which it will when we just started compass such as in the time to first query test) this timeout shouldn't matter. If it doesn't (like in the second connect test onwards) it will add up to 5 seconds to the tests.

What happened was that 1 second wasn't enough on macOS every time. We could try 2 seconds, I suppose, but it took so long to run every time to see if it helped that I got lazy :)

Reply:
Actually beforeTests is only run once for all of the smoke tests, so I can't think of a case where it wouldn't exist eventually.